scripts/fork-e2e-assert-json.mjs

```
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+
+import { readFileSync } from 'node:fs';
+
+const [mode, path, resourceType] = process.argv.slice(2);
```

ESLint

[error] 'process' is not defined.

(no-undef)